### PR TITLE
Add new Relief technique Card Style

### PR DESCRIPTION
**Pull request by AlexShor0103**
# CARD RESTYLING PR:
This PR is focused on adding some documentation, as well as mainly restyling our basic cards when displaying relief techniques

#### MAIN ADDITIONS:
- Added Gradients as constants
- Added getters for gradients based on name and mood
- Added font "Reem Kufi Fun" as main font, accessible and editable in pubspec.yaml
- Restyled cards in order to show gradient on top and have text + duration over white background

#### MAIN BUGS:
- Currently struggling with overflow of icon button on certain cards (see Talking To Someone card in screenshot)
- Text is automatically centered, rather than aligning left. Should be fixable by restructuring section of Relief Technique widget tree
- Not entirely sure where the color option is for the bottom nav bar

# Preview:
Screenshot 2022-09-20 164422

### CODE REVIEW

**Comment by trunci** [comment]
@AlexShor0103 didn't we agree that we would remove the different widths of the cards? I'm happy to do it in a different PR/commit, just wanted to make sure I got the idea right.

File: lib/components/ReliefActivitiesMenu/ActivityButton.dart [hunk 1]
```diff
@@ -6,6 +8,7 @@ import '../ReliefActivity/ReliefRateScreen.dart';
 import '../ReliefActivity/ReliefScreen.dart';
 import '../../utils/relief_technique_utils.dart';
 
+// this class
```
Comment (trunci) [inline lib/components/ReliefActivitiesMenu/ActivityButton.dart:11]: What is this supposed to represent?

Comment (trunci) [inline lib/components/ReliefActivitiesMenu/ActivityButton.dart]: What is this supposed to represent?

File: lib/components/ReliefActivitiesMenu/ActivityButton.dart [hunk 2]
```diff
@@ -20,53 +23,88 @@ class _ActivityButtonState extends State<ActivityButton> {
...
-                  iconData,
-                  size: 20,
-                )),
-            const Spacer(),
-          ]),
-          alignment: Alignment.center,
-        ),
         style: ButtonStyle(
-            backgroundColor: MaterialStateProperty.all<Color>(color),
+            //sets background color of buttons to white
```
Comment (trunci) [inline lib/components/ReliefActivitiesMenu/ActivityButton.dart:44]: Are those comments necessary?

File: lib/components/ReliefActivitiesMenu/ActivityButton.dart [hunk 3]
```diff
@@ -20,53 +23,88 @@ class _ActivityButtonState extends State<ActivityButton> {
...
+            backgroundColor: MaterialStateProperty.all<Color>(
+                Color.fromRGBO(255, 255, 255, 1)),
```
Comment (trunci) [inline lib/components/ReliefActivitiesMenu/ActivityButton.dart]: Can't you use the color from our App Constants?

File: lib/components/ReliefActivitiesMenu/ActivityButton.dart (hunk 2 above)
Comment (trunci) [reply lib/components/ReliefActivitiesMenu/ActivityButton.dart:44]: Valid for all comments in this PR found in the code.

File: lib/utils/constants.dart [hunk 4]
```diff
@@ -8,18 +8,51 @@ class AppConstants {
     sortingOptions.optionNotifier.value = sortOption;
   }
 
+  // static final colorLib ={
```
Comment (trunci) [inline lib/utils/constants.dart]: Please remove any unused commented out code.

**Comment by AlexShor0103** [comment]
@trunci Correct! I was planning on working on that in a different PR!

File: lib/utils/constants.dart [hunk 5]
```diff
@@ -8,18 +8,51 @@ class AppConstants {
...
+    'other': AppGrads.green
+  };
+
   static Color getColorByMood(String mood) {
     var color = colorsByMood[mood.toLowerCase()];
     return color ?? const Color(0x0);
   }
 
+  /* I had some issues when I tried to build the system same to the getColorsByMood
+  function, so I just made it a switch statement for now.
```
Comment (trunci) [inline lib/utils/constants.dart]: Please create an issue to fix this in the future.

**Review by trunci — CHANGES_REQUESTED** [review]
A few comments asking for mostly stylistic changes.

**Comment by trunci** [comment]
@AlexShor0103 do we have an issue for that yet? Feel free to link it here if so, else please create one. Thanks!

File: lib/utils/constants.dart (hunk 5 above)
Comment (AlexShor0103) [reply lib/utils/constants.dart]: Will do! Do you imagine that the way it's currently set up will cause a problem later on down the line? Or is it just better to have getColorByMood and get GradByMood to be similarly constructed?

**Comment by AlexShor0103** [comment]
We do not. I have added that as an issue!

File: lib/utils/constants.dart (hunk 4 above)
Comment (AlexShor0103) [reply lib/utils/constants.dart]: Will do!

File: lib/utils/constants.dart (hunk 5 above)
Comment (trunci) [reply lib/utils/constants.dart]: getGradByMood might be a better idea. We might need the fixed colors in the future too.

File: lib/components/ReliefActivitiesMenu/ActivityButton.dart (hunk 1 above)
Comment (AlexShor0103) [reply lib/components/ReliefActivitiesMenu/ActivityButton.dart]: That was an accident. I was looking to describe a class and then stopped midway through and when I went back to work on the code I didn't realize I left this in.

File: lib/components/ReliefActivitiesMenu/ActivityButton.dart (hunk 3 above)
Comment (AlexShor0103) [reply lib/components/ReliefActivitiesMenu/ActivityButton.dart]: Yes

File: lib/components/ReliefActivitiesMenu/ActivityButton.dart (hunk 2 above)
Comment (AlexShor0103) [reply lib/components/ReliefActivitiesMenu/ActivityButton.dart:44]: They aren't necessary! I was making some comments along the way as I was familiarizing myself with the code, and didn't get rid of them when making my PR

**Pull request merged**